package uk.tfindustries.blueaerotweaks.util;

import net.minecraft.resources.ResourceLocation;
import net.minecraft.tags.BlockTags;
import net.minecraft.tags.ItemTags;
import net.minecraft.tags.TagKey;
import net.minecraft.world.item.Item;
import net.minecraft.world.level.block.Block;
import uk.tfindustries.blueaerotweaks.BlueAeroTweaks;

public class ModTags {
    public static class Blocks {

        private static TagKey<Block> createTag(String name) {
            return BlockTags.create(ResourceLocation.fromNamespaceAndPath(BlueAeroTweaks.MODID, name));
        }
    }

    public static class Items {
        public static final TagKey<Item> SQUALL_ITEMS = createTag("squall_items");

        private static TagKey<Item> createTag(String name) {
            return ItemTags.create(ResourceLocation.fromNamespaceAndPath(BlueAeroTweaks.MODID, name));
        }
    }
}
